wordpress.org/public_html/wp-content/plugins/wporg-profiles-wp-activity-notifier/wporg-profiles-wp-activity-notifier.php

```diff
@@ -186,24 +230,24 @@ public function notify_new_blog_post( $post ) {
 	/**
 	 * Handler for comment creation.
 	 *
-	 * @param int        $id      Comment ID
-	 * @param WP_Comment $comment Comment
+	 * @param int        $id      Comment ID.
+	 * @param WP_Comment $comment Comment.
 	 */
-	function insert_comment( $id, $comment ) {
-		if ( 1 == $comment->comment_approved ) {
+	public function insert_comment( $id, $comment ) {
+		if ( 1 === $comment->comment_approved ) {
```

Comment:
> Make sure you test changes like this if you haven't already, because WP APIs sometimes return ints as strings, etc.

Wasn't aware of this fact, thanks for the reminder, I've reverted it.

> I'll often only run phpcbf on old code, because it will make the formatting consistent without making any changes that are potentially breaking (and therefore require testing).

Good point, thanks! 👍 

> Similarly, renaming functions needs testing too, in case some of the callers were missed, or maybe there are child classes that need updating too, etc.

The functions I've renamed in this PR are `_notify_forum_topic_payload` and `_notify_forum_forum_payload`. I've grep them in sandbox and they are only used in the file I renamed them. It seems safe to me so I keep the change.

Reply:
👍🏻